src/fairchem/core/README.md

```diff
@@ -1,59 +1,28 @@
-# `fairchem-core` by Open Catalyst Project
+<h1 align="center"> <code>fairchem</code> by FAIR Chemistry </h1>
+
+<p align="center">
+  <img width="559" height="200" src="https://github.com/FAIR-Chem/fairchem/assets/45150244/5872c21c-8f39-41af-b703-af9817f0affe"?
+</p>
 
-![tests](https://github.com/FAIR-Chem/fairchem/actions/workflows/test.yml/badge.svg?branch=main)
 
-`fairchem-core` is the [Open Catalyst Project](https://opencatalystproject.org/)'s
-library of state-of-the-art machine learning algorithms for catalysis.
+<h4 align="center">
 
-<div align="left">
-    <img src="https://user-images.githubusercontent.com/1156489/170388229-642c6619-dece-4c88-85ef-b46f4d5f1031.gif">
-</div>
+![tests](https://github.com/FAIR-Chem/fairchem/actions/workflows/test.yml/badge.svg?branch=main)
+[![documentation](https://github.com/FAIR-Chem/fairchem/actions/workflows/docs.yml/badge.svg?branch=main)](https://github.com/FAIR-Chem/fairchem/actions/workflows/docs.yml)
+[![Static Badge](https://img.shields.io/badge/python-3.9%2B-blue)](https://www.python.org/downloads/)
 
-## [OCP Documentation](https://fair-chem.github.io/)
-Full documentation, released data/checkpoints, and tutorials are now available.
+</h4>
 
-- [Installation](https://fair-chem.github.io/core/install.html)
-- [Pretrained models](https://fair-chem.github.io/core/models.html)
-- [FAQ](https://fair-chem.github.io/core/model_faq.html)
+`fairchem` is the [FAIR](https://ai.meta.com/research/) Chemistry's centralized repository of all its data, models, demos, and application efforts for materials science and quantum chemistry.
 
-`ocp` provides training and evaluation code for tasks and models that take arbitrary
+`fairchem` provides training and evaluation code for tasks and models that take arbitrary
 chemical structures as input to predict energies / forces / positions / stresses,
 and can be used as a base scaffold for research projects. For an overview of
 tasks, data, and metrics, please read the documentations and respective papers:
  - [OC20](https://fair-chem.github.io/core/datasets/oc20.html)
  - [OC22](https://fair-chem.github.io/core/datasets/oc22.html)
  - [ODAC23](https://fair-chem.github.io/core/datasets/odac.html)
 
-Projects and models built on `ocp`:
```

Comment:
Is there a reason we dont want these references anymore?

Reply:
You're right that makes more sense! I was going by just the docs for the pypi-publish action, but according to Github - that url is used to link the deployment that shows up on github

Reply:
Not really - I just trimmed the README (which is the one that will show up on PyPi) since most of that information is in the docs now.

Happy to add anything that we want to show up on pypi